manifests/filter/container.pp

```diff
@@ -311,8 +323,10 @@
   $syslog_port                          = $repose::params::syslog_port,
   $syslog_protocol                      = $repose::params::syslog_protocol,
   $via                                  = undef,
+  $via_header                           = {},
   $flume_host                           = $repose::params::flume_host,
   $flume_port                           = $repose::params::flume_port,
+  $repose_version                       = 'true',
```

Comment:
Not certain what this is doing?

Reply:
Thanks, good catch, missed an unused option. Removed in amended commit.